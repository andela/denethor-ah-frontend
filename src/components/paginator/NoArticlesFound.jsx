import React from 'react';
import { library } from '@fortawesome/fontawesome-svg-core';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { faTimesCircle } from '@fortawesome/free-regular-svg-icons';

library.add(faTimesCircle);

const NoArticlesFound = () => (
  <div className='flex search-error'>
    <FontAwesomeIcon
      icon={faTimesCircle}
      size='3x'
      color='#ff0000'
    />
    &nbsp;&nbsp;&nbsp;
    <h2>No Article was found in this category</h2>
  </div>
);

export default NoArticlesFound;
