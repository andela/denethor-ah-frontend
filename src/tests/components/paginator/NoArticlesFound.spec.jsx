import React from 'react';
import { shallow } from 'enzyme';
import NoArticlesFound from '../../../components/paginator/NoArticlesFound';

test('Should render static component', () => {
  const wrapper = shallow(<NoArticlesFound />);
  expect(wrapper).toMatchSnapshot();
});
