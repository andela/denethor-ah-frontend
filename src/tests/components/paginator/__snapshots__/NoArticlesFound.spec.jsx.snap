// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render static component 1`] = `
<div
  className="flex search-error"
>
  <FontAwesomeIcon
    border={false}
    className=""
    color="#ff0000"
    fixedWidth={false}
    flip={null}
    icon={
      Object {
        "icon": Array [
          512,
          512,
          Array [],
          "f057",
          "M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm0 448c-110.5 0-200-89.5-200-200S145.5 56 256 56s200 89.5 200 200-89.5 200-200 200zm101.8-262.2L295.6 256l62.2 62.2c4.7 4.7 4.7 12.3 0 17l-22.6 22.6c-4.7 4.7-12.3 4.7-17 0L256 295.6l-62.2 62.2c-4.7 4.7-12.3 4.7-17 0l-22.6-22.6c-4.7-4.7-4.7-12.3 0-17l62.2-62.2-62.2-62.2c-4.7-4.7-4.7-12.3 0-17l22.6-22.6c4.7-4.7 12.3-4.7 17 0l62.2 62.2 62.2-62.2c4.7-4.7 12.3-4.7 17 0l22.6 22.6c4.7 4.7 4.7 12.3 0 17z",
        ],
        "iconName": "times-circle",
        "prefix": "far",
      }
    }
    inverse={false}
    listItem={false}
    mask={null}
    pull={null}
    pulse={false}
    rotation={null}
    size="3x"
    spin={false}
    symbol={false}
    title=""
    transform={null}
  />
     
  <h2>
    No Article was found in this category
  </h2>
</div>
`;
